specs/permissions-deployment-schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "type": "object",
+    "properties": {
+        "permissionDeployments": {
+            "type": "object",
+            "patternProperties": {
+                "[\\w]+\\.[\\w]+[\\.[\\w]+]?": {
+                    "type": "object",
+                    "properties": {
+                        "id": {
+                            "type": "string"
+                        },
+                        "resourceAppId": {
+                            "type": "string"
+                        },
+                        "environment": {
+                            "type": "string"
```

Comment:
Have we defined valid values for the `environment`?

Reply:
We can open a separate PR for that once we know what the value values will be.  These values are fairly dynamic so I would suggest that we don't use an enum in the implementation.